cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -1569,7 +1594,7 @@ viewTypeLabel theme moduleName typeName typeDef docs =
                         else
                             "one of"
                 )
-                backgroundColor
+                Theme.defaultColors.backgroundColor
```

Comment:
This refers to the `defaultColors` which is static and won't change if we change the theme. I know `defaultColors` was already there but we should probably remove it and use the `theme.colors` everywhere instead.